ibrdtn/daemon/tests/unittests/FakeDatagramService.cpp

```diff
@@ -105,7 +105,7 @@ size_t FakeDatagramService::recvfrom(char *buf, size_t length, char &type, char
 		seqno = msg.seqno;
 		address = msg.address;
 		ret = msg.data.size();
-		::memcpy(buf, &msg.data[0], (ret > length) ? length : ret);
```

Comment:
BTW, C++98 does not guarantee vector buffer to be contiguous, that was only added in C++03.